test/manager/baseManagerV2.spec.ts

```diff
@@ -0,0 +1,1710 @@
+import "module-alias/register";
```

Comment:
These tests are amazing and so thorough. Well laid out and even checking to make sure all critical pieces of state remain unchanged within different flows. This is *chef's kiss*.